Makefile

```diff
@@ -384,7 +384,7 @@ build-docker-desmosnode:
 # Setups 4 folders representing each one the genesis state of a testnet node
 setup-localnet: build-linux
 	if ! [ -f build/node0/desmos/config/genesis.json ]; then $(BUILDDIR)/desmos testnet \
-		-o ./build --starting-ip-address 192.168.10.2 --keyring-backend=test \
+		-o ./build --starting-ip-address 192.168.255.2 --keyring-backend=test \
```

Comment:
This change, as well as other IP-related changes, were made in order to avoid any conflict if developers are connected to a nertwork that uses subnet `192.168.10.0`.